Classes/Views/LabelListView.swift

```diff
@@ -63,6 +67,18 @@ UICollectionViewDelegateFlowLayout {
         collectionView.frame = bounds
     }
 
+    override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+        guard let lastCell = collectionView.cellForItem(at: [0, labels.count - 1])
+            else { return self }
+        return point.x > lastCell.frame.maxX ? self : super.hitTest(point, with: event)
+    }
```

Comment:
<img width="305" alt="screenshot 2018-11-11 11 57 13" src="https://user-images.githubusercontent.com/26695477/48315820-1804c680-e5a9-11e8-930a-66e32175d3ba.png">

I highlighted the `LabelListView` frame in orange. The UicollectionView's frame is set to it's bounds. 

If you click inside the orange border on an area that's not a LabelCell, nothing happens. In the HitTest override, I'm trying to process taps that land on a label cell, and pass through taps to the right of the right-most label cell. 

Perhaps setting the CollectionView's frame is more appropriate, so that it's only side wide as it needs to be given the number of labels it presents?

Reply:
Fudging around, something like this seems to send the taps to the right places: 

```Swift
// this is copied/pasted from existing methods to find width 
private var labelTextTotalWidth: CGFloat {
        let interitemSpacing = labels.count > 1 ? CGFloat(labels.count - 1) * Styles.Sizes.labelSpacing : 0.0
        return labels.reduce(0, { $0 + LabelListCell.size($1.name).width }) + interitemSpacing
    }

override func layoutSubviews() {
        super.layoutSubviews()
        collectionView.frame = CGRect(
            x: bounds.minX,
            y: bounds.minY,
            width: labelTextTotalWidth,
            height: bounds.height
        )
}

```